account_analytic_parent/models/account_analytic_account.py

```diff
@@ -27,10 +27,25 @@ def _compute_debit_credit_balance(self):
         of analytic account changes
         """
         super(AccountAnalyticAccount, self)._compute_debit_credit_balance()
-        for account in self:
-            account.debit += sum(account.mapped('child_ids.debit'))
-            account.credit += sum(account.mapped('child_ids.credit'))
-            account.balance += sum(account.mapped('child_ids.balance'))
+        analytic_line_obj = self.env['account.analytic.line']
+        # compute only analytic line
+        for account in self.filtered(lambda x: x.child_ids):
+            domain = [('account_id', 'in', account.child_ids.ids)]
```

Comment:
operator should be `child_of` for including account itself.

Reply:
and do the test over `self.ids`, not `self.child_ids.ids`

Reply:
account itself is not included in the loop but it was already included when calling super, then in the loop we increment the values by adding the children ones. But maybe is less confusing doing that way, it's ok.

Reply:
This is resolved, isn't it?

Reply:
@pedrobaeza Can you please update your review? IMO your comment is attended.